cmd/firetron/fetcher.go

```diff
@@ -62,7 +62,10 @@ func fetchE(cmd *cobra.Command, args []string) error {
 	tronClients := firecoreRPC.NewClients(maxBlockFetchDuration, rollingStrategy, logger)
 
 	for _, endpoint := range rpcEndpoints {
-		client := rpc.NewTronClient(endpoint, apiKey)
+		client, err := rpc.NewTronClient(endpoint, apiKey)
+		if err != nil {
+			return fmt.Errorf("failed to create Tron client for endpoint %q: %w", endpoint, err)
+		}
```

Comment:
Since I added error propagation on `NewTronClient,` do we want to fail here, or should we just output the ignore and maybe just fail afterwards if we have no client?

Or is that something already handled?

Reply:
All good.